<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphDesignerPage</class>
 <widget class="QMainWindow" name="GraphDesignerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolbar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="acImageExporter"/>
   <addaction name="acNodes_Exporter"/>
   <addaction name="acGraphSettings"/>
   <addaction name="separator"/>
   <addaction name="acEdgeAttrs"/>
   <addaction name="acNodeAttrs"/>
   <addaction name="acGraphGen"/>
  </widget>
  <action name="acImageExporter">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/material-icons/image/png/ic_photo_camera_white_24dp.png</normaloff>:/icons/material/material-icons/image/png/ic_photo_camera_white_24dp.png</iconset>
   </property>
   <property name="text">
    <string> Image Exporter</string>
   </property>
   <property name="toolTip">
    <string>Image Exporter</string>
   </property>
  </action>
  <action name="acNodes_Exporter">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/material-icons/editor/png/baseline_table_chart_white_24dp.png</normaloff>:/icons/material/material-icons/editor/png/baseline_table_chart_white_24dp.png</iconset>
   </property>
   <property name="text">
    <string>Nodes Exporter</string>
   </property>
   <property name="toolTip">
    <string>Nodes Exporter</string>
   </property>
  </action>
  <action name="acGraphSettings">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/settings_white_24</normaloff>:/icons/material/settings_white_24</iconset>
   </property>
   <property name="text">
    <string>Graph Settings</string>
   </property>
   <property name="toolTip">
    <string>Graph Settings</string>
   </property>
  </action>
  <action name="acEdgeAttrs">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</iconset>
   </property>
   <property name="text">
    <string>Edge Attributes Editor</string>
   </property>
   <property name="toolTip">
    <string>Edge Attributes Editor</string>
   </property>
  </action>
  <action name="acNodeAttrs">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</iconset>
   </property>
   <property name="text">
    <string>Node Attributes Editor</string>
   </property>
   <property name="toolTip">
    <string>Node Attributes Editor</string>
   </property>
  </action>
  <action name="acGraphGen">
   <property name="icon">
    <iconset resource="../res/guiRes.qrc">
     <normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</normaloff>:/icons/material/material-icons/toggle/svg/ic_radio_button_unchecked_24px.svg</iconset>
   </property>
   <property name="text">
    <string>Graph Generator</string>
   </property>
   <property name="toolTip">
    <string>Graph Generator</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/guiRes.qrc"/>
 </resources>
 <connections/>
</ui>
